src/contracts/atlas/Storage.sol

```diff
@@ -22,7 +22,8 @@ contract Storage is AtlasEvents, AtlasErrors, AtlasConstants {
     uint8 public constant decimals = 18;
 
     // Gas Accounting public constants
-    uint256 public constant SURCHARGE_RATE = 1_000_000; // 1_000_000 / 10_000_000 = 10%
+    uint256 public constant ATLAS_SURCHARGE_RATE = 1_000_000; // 1_000_000 / 10_000_000 = 10%
+    uint256 public constant BUNDLER_SURCHARGE_RATE = 1_000_000; // 1_000_000 / 10_000_000 = 10%
     uint256 public constant SURCHARGE_SCALE = 10_000_000; // 10_000_000 / 10_000_000 = 100%
     uint256 public constant FIXED_GAS_OFFSET = 100_000;
```

Comment:
`FIXED_GAS_OFFSET` may need to be adjusted up - looks like there's a lot more logic and therefore gas cost happening after the ending `gasleft()` snapshot is taken, than there was when Riley calculated this during the audit.

Reply:
Good catch.

Reply:
We should actually re-calculate it after the transient storage PR. Won't be accurate yet.